lib/html/HtmlParser.js

```
@@ -25,6 +25,7 @@ const {
 	webpackCommentRegExp
 } = require("../util/magicComment");
 const memoize = require("../util/memoize");
+const { JAVASCRIPT_SCRIPT_TYPES } = require("./data");
```

ESLint

[error] Missing file extension for "./data"

(import/extensions)